Lib/test/test_fstring.py

```diff
@@ -329,13 +334,13 @@ def test_ast_line_numbers_multiline_fstring(self):
         self.assertEqual(t.body[1].lineno, 3)
         self.assertEqual(t.body[1].value.lineno, 3)
         self.assertEqual(t.body[1].value.values[0].lineno, 3)
-        self.assertEqual(t.body[1].value.values[1].lineno, 3)
-        self.assertEqual(t.body[1].value.values[2].lineno, 3)
+        # TODO: RUSTPYTHON self.assertEqual(t.body[1].value.values[1].lineno, 4)
```

Comment:
we usually do not comment out test lines, but turn off the test itself.
Could you revert this part and mark `test_ast_line_numbers_multiline_fstring` as expectedFailure?